parent/pom.xml

```diff
@@ -1653,6 +1655,11 @@
                 <artifactId>encoder</artifactId>
                 <version>${owasp.encoder.wso2.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.orbit.commons-text</groupId>
+                <artifactId>commons-text</artifactId>
+                <version>${commons-text.version}</version>
```

Comment:
Are we referring to `commons-text.orbit.version` version here ?

Reply:
Yes, we don't need this dependency since the p2 plugin itself is resolving dependencies.